src/lib.rs

```diff
@@ -208,6 +208,8 @@ pub trait Context {
 
     unsafe fn buffer_data_u8_slice(&self, target: u32, data: &[u8], usage: u32);
 
+    unsafe fn buffer_sub_data(&self, target: u32, offset: i32, src_data: &[u8]);
```

Comment:
Could we match the naming of `buffer_data_u8_slice`, like `buffer_sub_data_u8_slice`?

Reply:
Sure thing, I didn't know if you wanted the same naming convention even though there's only the one method.